src/parsers/strategies.rs

```diff
@@ -148,25 +148,36 @@ fn create_simple_column(column_name: &str, table_name: &str) -> SimpleColumn {
     }
 }
 
-fn column_transformer_is_overriden(
+fn apply_transformer_overrides(
```

Comment:
Do you think it's worth adding some tests for the new scramble option? To go with the existing tests for the commercially sensitive and allow potentially pii flags

Reply:
Good idea, I'll add some

Reply:
I've added a test and modified an existing one as well